packages/protocol/src/action/protocols.ts

```diff
@@ -108,10 +122,20 @@ export interface ClusterTypeProtocol extends AddressableElementProtocol<ClusterI
      */
     attributes: CollectionProtocol<AttributeTypeProtocol>;
 
+    /**
+     * Map of attribute names to IDs.
+     */
+    attributeNameToId: Map<string, AttributeId>;
+
     /**
      * Event metadata.
      */
     events: CollectionProtocol<EventTypeProtocol>;
+
+    /**
+     * Map of event names to IDs.
+     */
+    eventNameToId: Map<string, EventId>;
```

Comment:
Maybe add a TODO noting that we need this temporarily while events come in through the OccurrenceManager.

Longer term I think we should convert OccurrenceManager to serialize IDs.

Then decouple OccurrenceManager from @matter/node... Instead have another Observable on NodeProtocol for events, and then have NodeProtocol pass them to OccurrenceManager.

That would simplify the contract between @matter/node and @matter/protocol and also be more efficient.

Reply:
likely eventToName is not needed ... i an remove that ...

Reply:
eventToName removed